tankersdk_identity/__init__.py

```diff
@@ -79,8 +79,16 @@ def _serialize_identity(identity: Dict[str, Any]) -> str:
     return base64.b64encode(as_json.encode()).decode()
 
 
-def upgrade_identity(identity: str) -> str:
-    return _serialize_identity(_deserialize_identity(identity))
+def upgrade_identity(identityB64: str) -> str:
+    identity = _deserialize_identity(identityB64)
+    if identity["target"] == "email":
```

Comment:
You want to ensure the identity is a public one

Reply:
Thanks! I forget every time =)